Dockerfile.centos

```diff
@@ -4,8 +4,9 @@ RUN yum module -y install go-toolset postgresql && yum -y install git
 ENV GOPATH=/go
 ENV GO111MODULE=on
 
-ADD go.mod  /go/src/app/
-ADD go.sum  /go/src/app/
+ADD /_generated /go/src/app/_generated
```

Comment:
As `_generated` folder is not included in repo after pull, it breaks many things that worked till now (docker-compose, go tests, docker build). When generating is not working then nothing is working.

Reply:
Fixed with the dedicated repository for clients.